docs/additional-examples/evm-swap.md

```diff
@@ -0,0 +1,658 @@
+*/
+```
+:::info
+You can use any of the chains [supported](../resources/supported-chains) by Lit when setting your own swap parameters
+:::
+
+The following code can be used to generate the swap Lit Action based on the parameters above: [swapActionGenerator.js](https://github.com/anshss/lit-evm-swap/blob/main/lit/swapActionGenerator.js)
+
+Remember, once a Mint/Grant/Burn PKP is created, anyone can execute the Lit Action on the Lit Network. This execution needs to define the parameters needed by the Lit Action to run. While generating a sign or checking conditions on the action, we need information around the pkp for which we are doing it. Due to this, we need to construct our Lit Action in a way that works with any PKP.
+
+Our Lit Action primarily focuses on 4 factors:
+
+- Access control conditions (ACCs) to ensure that funds have been received on both chains
+- Transaction objects for transferring funds between our two swap parties
+- Clawback transaction objects to revert transferred funds back to their owners if the swap parameters aren't met
+- [Lit action conditions](../sdk/access-control/lit-action-conditions.md) to determine which transaction objects should be signed and returned
+
+We'll use an [Access Control Condition (ACC)](../sdk/access-control/evm/basic-examples#must-posess-at-least-one-erc20-token) here to verify if the swap conditions are met before it is executed. These conditions are:
+
+- Have token A funds reached on Chain A?
+- Have token B funds reached on Chain B?
+
+If yes, then forward the funds and complete the swap or else revert them.
+
+A condition on Base Sepolia for checking if the PKP contains the exact amount of token A in Wei will look like this:
+
+```js
+const chainACondition = {
+    conditionType: "evmBasic",
+    contractAddress: "0xad50f302a957C165d865eD398fC3ca5A5A2cDA85",
+    standardContractType: "ERC20",
+    chain: "baseSepolia",
+    method: "balanceOf",
+    parameters: ["address"],
+    returnValueTest: { comparator: "==", value: "4000000000000000000" },
+};
+// parameter field would be later replaced by the pkpAddress in the action
+```
+We need to create two transaction objects, each of which will transfer the funds between the two swap parties that have been set. Each party will call the `transfer(address, uint256)` function on the token's smart contract. This information is captured in the "data" field of the object which we can fetch in the following way:
+
+```js
+function generateCallData(counterParty, amount) {
+    const transferInterface = new ethers.utils.Interface([
+        "function transfer(address, uint256) returns (bool)",
+    ]);
+    return transferInterface.encodeFunctionData("transfer", [
+        counterParty,
+        amount,
+    ]);
+}
+```
+
+A transaction object would then look something like below
+
+```js
+let chainATransaction = {
+    to: "0xad50f302a957C165d865eD398fC3ca5A5A2cDA85",
+    gasLimit: "60000",
+    from: "0x291B0E3aA139b2bC9Ebd92168575b5c6bAD5236C",
```

Comment:
I think you could just omit this property. We don't usually specify the `from` property in an ETH transaction since it's derived from the signature